### PR TITLE
Pano showfps, showpos, and showbonecounts panels

### PR DESCRIPTION
**Pull request by braem**
Closes https://github.com/momentum-mod/game/issues/1516

Merge with red PR

### CODE REVIEW

File: styles/components/showfps.scss [hunk 1]
```diff
@@ -0,0 +1,34 @@
+@use '../config' as *;
+@use '../abstract/mixin';
+
+.fpsmeter {
+	width: 100%;
```
Comment (tsa96) [inline styles/components/showfps.scss:5]: Does this need to be `width: 100%`/`height: 100%`? Even if it's top-level can't we just position without that?

**Review by tsa96 — CHANGES_REQUESTED** [review]

File: styles/components/showfps.scss [hunk 2]
```diff
@@ -0,0 +1,34 @@
...
+	height: 100%;
+	align: left top;
+
+	&__label {
+		@include mixin.font-styles($use-header: true);
+		font-size: 16px;
+		text-align: left;
+		text-overflow: ellipsis;
```
Comment (PeenScreeker) [inline styles/components/showfps.scss:13]: not sure if it matters, `ellipsis` is the default mode for `text-overflow`.

**Review by PeenScreeker — APPROVED** [review]

File: styles/components/showfps.scss (hunk 1 above)
Comment (braem) [reply styles/components/showfps.scss:5]: We can yeah, if we ever want other panels below it. I'd like it to be absolutely positioned in a place that makes more sense for UI in the future

Comment (Gocnak) [reply styles/components/showfps.scss:5]: Yeah I think this is fine until we do more absolute positioning (HUD customization)

**Review by Gocnak — APPROVED** [review]

**Pull request merged**